docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+```
+peer channel update -f channel-artifacts/config_update_in_envelope.pb -c channel1 -o localhost:7050  --ordererTLSHostnameOverride orderer.example.com --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+
+When the channel update is successful, you should see the following response:
+```
+2020-01-09 21:30:45.791 UTC [channelCmd] update -> INFO 002 Successfully submitted channel update
+```
+
+We can set the anchor peers for Org2. Because we are going through the process a second time, we will go through the steps more quickly. Set the environment variables to operate the `peer` CLI as the Org2 admin:
+```
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_LOCALMSPID="Org2MSP"
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
+export CORE_PEER_ADDRESS=localhost:9051
+```
+
+Pull the latest channel configuration block, which is now the second block on the channel:
+```
+peer channel fetch config channel-artifacts/config_block.pb -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com -c channel1 --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+
+Navigate back to the `channel-artifacts` directory:
+```
+cd channel-artifacts
+```
+
+You can then decode and copy the configuration block.
+```
+configtxlator proto_decode --input config_block.pb --type common.Block --output config_block.json
+jq .data.data[0].payload.data.config config_block.json > config.json
+cp config.json config_copy.json
+```
+
+Add the Org2 peer that is joined to the channel as the anchor peer in the channel configuration:
+```
+jq '.channel_group.groups.Application.groups.Org2MSP.values += {"AnchorPeers":{"mod_policy": "Admins","value":{"anchor_peers": [{"host": "peer0.org2.example.com","port": 9051}]},"version": "0"}}' config_copy.json > modified_config.json
+```
+
+We can now convert both the original and updated channel configurations back into protobuf format and calculate the difference between them.
+```
+configtxlator proto_encode --input config.json --type common.Config --output config.pb
+configtxlator proto_encode --input modified_config.json --type common.Config --output modified_config.pb
+configtxlator compute_update --channel_id channel1 --original config.pb --updated modified_config.pb --output config_update.pb
+```
+
+Add the metadata back to the channel configuration update to create the new configuration block that we can use to update the channel:
```

Comment:
I thought I already left a comment on this, but it seems to be missing.

But here, we are not 'adding metadata back', and we are certainly not creating a config block.  We are wrapping the config update into a `common.Envelope` which is the Fabric transaction encoding.  So, we are creating a config update transaction.

Reply:
How about: We can wrap the configuration update in a transaction envelope to create the channel configuration update transaction.

Reply:
Works for me.

Reply:
This isn't resolved yet.  Still needs to be fixed.

Reply:
There were two references to metadata. I remove the second one and replaced with:
Wrap the configuration update in a transaction envelope to create the channel configuration update transaction: